modules/gapi/include/opencv2/gapi/infer/onnx.hpp

```diff
@@ -351,6 +351,7 @@ struct ParamDesc {
     std::unordered_map<std::string, std::pair<cv::Scalar, cv::Scalar> > generic_mstd;
     std::unordered_map<std::string, bool> generic_norm;
 
+    std::unordered_map<std::string, std::string> session_options_map;
```

Comment:
should be just `map` otherwise won't work with python bindings

Reply:
Yes I remember now. Fixing it.